spec/requests/api/v2/people_controller_update_spec.rb

```diff
@@ -0,0 +1,189 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V2::PeopleController do
+  let(:response_json) { JSON.parse(response.body) }
+  let(:access_token) { create(:access_token).token }
+  let(:content_type) { ApiController::CONTENT_TYPE }
+  let(:headers) { { 'CONTENT_TYPE': content_type }.merge('Authorization' => "Bearer #{access_token}") }
+
+  describe 'PUT /api/v2/people' do
+    let(:schema) { load_yaml_schema('put_people_responses.yaml') } # Note: V1 put_people_responses is equal V2 put_people_responses
```

Comment:
I would suggest we add a new `patch_people_responses.yaml` in v2, this way we fully separate the implementations, even if some yaml files are similar.

Reply:
good point, I think its PATCH if you can update some attributes of the object, or PUT if you are replacing the entire object. I'd suggest sticking with PATCH as we use that elsewhere.

Reply:
I agree with you that we need to be consistent,
in this case, I would stick to PUT, since the `/api/v1/people` is using PUT,
and would be strange to have v1 with PUT and v2 with PATCH.

(Also consider that both put and patch point to the same update action in controller.)

Reply:
So, reading over https://en.wikipedia.org/wiki/Patch_verb#:~:text=The%20main%20difference%20between%20the,instructions%20to%20modify%20the%20resource. I definitely think this should be PATCH - as you allow the user to update a single attribute rather than the whole object

Reply:
Update: yes, actually I can see the Doc states `PATCH` but spec uses PUT (both are supported anyway),
I am happy to change the test to patch (since put will still continue to work)

Reply:
update2: Changed PUT to PATCH, as suggested.